packages/forklift-console-plugin/src/modules/Providers/views/create/components/VCenterProviderCreateForm.tsx

```diff
@@ -101,18 +95,12 @@ export const VCenterProviderCreateForm: React.FC<VCenterProviderCreateFormProps>
       if (id == 'sdkEndpoint') {
         const sdkEndpoint = trimmedValue || undefined;
 
-        // Revalidate URL - VCenter or ESXi
```

Comment:
nice refactor, once we separated the pages of esxi and vcenter we don't need this test enymore.